packages/patrol_cli/lib/src/analytics/analytics.dart

```diff
@@ -85,7 +86,13 @@ class Analytics {
     );
   }
 
-  bool get enabled => _config?.enabled ?? false;
+  bool get enabled {
+    if (ci.isCI) {
+      return false;
+    }
```

Comment:
Do we also want to track commands executed on CI (because we could by attaching `is_ci: bool` in event data.